1.2/Dockerfile

```diff
@@ -12,7 +12,13 @@ ENV NPM_VERSION ${NPM_VERSION}
 LABEL io.k8s.description="Platform for building and running a phoenix app" \
       io.k8s.display-name="build-phoenix" \
       io.openshift.expose-services="4000:http" \
-      io.openshift.tags="builder,elixir,phoenix"
+      io.openshift.tags="builder,elixir,phoenix" \
+      com.redhat.dev-mode="DEV_MODE:false" \
+      com.redhat.deployments-dir="/opt/app-root/src" \
+      com.redhat.dev-mode.port="DEBUG_PORT:5858" \
```

Comment:
What is the purpose of the debug port? There is nothing listening on that port...